firmware/integration/rusefi_config.txt

```diff
@@ -1119,7 +1119,19 @@ int16_t tps2Max;Full throttle#2. tpsMax value as 10 bit ADC value. Not Voltage!\
 	uint8_t un1used_former_warmup_target_afr;;"units", 1, 0, -20, 100, 0
 	can_baudrate_e can2BaudRate;
 
-	uint8_t[18] unused_former_warmup_target_afr;;"units", 1, 0, -20, 100, 0
+	uint32_t verboseCan2BaseAddress;;"", 1, 0, 0, 536870911, 0
+	bit enableVerboseCan2Tx;+CAN broadcast using custom rusEFI protocol\nenable can_broadcast/disable can_broadcast
+	bit can2ReadEnabled;enable can_read/disable can_read
+	bit can2WriteEnabled;enable can_write/disable can_write
+	can_nbc_e can2NbcType;set can_mode X
```

Comment:
do we want to be able to transmit dash on both busses at once? do we need that? or just the option to select which bus it happens on?

Reply:
Good point, I can't think of a situation where you need dash xmit on both busses.